RenderSystems/GLSupport/src/OSX/OgreOSXCocoaWindow.mm

```diff
@@ -128,6 +128,12 @@ - (BOOL)acceptsFirstResponder
 
         ***Key: "currentGLContext" Description: use an externally created OpenGL context (must be current)
          Values: true, false Default: false
+
```

Comment:
no need to duplicate the documentation hre

Reply:
Ok, I removed it.